retrofit2-clients/src/test/java/com/palantir/remoting3/retrofit2/Retrofit2ClientApiTest.java

```diff
@@ -71,9 +71,9 @@ public void testOptionalStringHandling() throws IOException, InterruptedExceptio
 
         server.enqueue(new MockResponse().setBody("\"pong\""));
         assertThat(service.getGuavaOptionalString(guavaOptional("p"), guavaEmptyOptional()).execute().body())
-            .isEqualTo(guavaOptional("pong"));
+                .isEqualTo(guavaOptional("pong"));
         assertThat(server.takeRequest().getPath())
-            .isEqualTo("/getGuavaOptionalString/p/");
+                .isEqualTo("/getGuavaOptionalString/p/");
 
```

Comment:
@rfink could you copy in the extra tests I wrote to cover retrofit error handling?  It was pretty under-specified before: https://github.com/palantir/http-remoting/pull/623/files#diff-506462de9dcfed10a388f16cf93d6c96R136

- completableFuture_should_throw_RemoteException_for_server_serializable_errors
- sync_retrofit_call_should_throw_RemoteException_for_server_serializable_errors
- async_retrofit_call_should_throw_RemoteException_for_server_serializable_errors
- completableFuture_should_throw_normal_IoException_for_client_side_errors

Reply:
done, except for async_retrofit_call_should_throw_RemoteException_for_server_serializable_errors , see https://github.com/palantir/http-remoting/issues/625